regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -123,23 +123,13 @@ var CommentReviewView = Backbone.View.extend({
   },
 
   initToggles: function() {
-    function toggle($elm) {
-      var $target = $('#' + $elm.data('toggle'));
-      if ($target.is(':visible')) {
-        $target.hide();
-        $elm.html($target.data('more-text') || '<span class="fa fa-plus-circle text-expand" aria-hidden="true"></span>Show more');
-      } else {
-        $target.show();
-        $elm.html($target.data('less-text') || '<span class="fa fa-minus-circle text-expand" aria-hidden="true"></span>Show less');
-      }
-    }
-    var $toggles = this.$el.find('[data-toggle]');
-    $toggles.each(function(idx, elm) {
-      toggle($(elm));
-    });
-    $toggles.on('click', function() {
-      toggle($(this));
+
+    this.$el.find('.show-more-toggle').click(function () {
+      $(this).find('.show-more-toggle-open').toggle();
+      $(this).find('.show-more-toggle-close').toggle();
+      $(this).prev().toggle();
```

Comment:
This is much more compact, but using `prev()` couples the logic more tightly to the markup. Are we sure we want to do that?